robotd/master.py

```diff
@@ -11,12 +11,23 @@
 import pyudev
 import setproctitle
 
-from robotd.devices import BOARDS
+import textwrap
+
+from robotd.devices import BOARDS, STARTUP_BOARDS
+
+
+def _send(connection, message):
+    # Split the message into buf_size chunks
+    buf_size = BoardRunner.SOCK_BUFFER_SIZE
+    for msg in [message[i:i + buf_size] for i in range(0, len(message), buf_size)]:
+        connection.send(msg)
```

Comment:
I'm trying to avoid calling recv(524288) as that's a big buffer size. From my testing, if you ever call recv() on something smaller than the size of the sent packet, data is lost. Therefore we need to enforce a maximum packet size on the sending end